docs/connectors/sources/debezium-mysql.md

```diff
@@ -3,5 +3,160 @@
+which provides more detailed information and notes regarding Debezium connectors.
+
+
+## Step 1: Enable binlog
+
+Binary logging with row-based format must be enabled.
+
+
+### Generic: MySQL
+
+MySQL version 8.0 has binary logging enabled by default.
+As such, no settings have to be adjusted unless your database
+is configured differently than the default.
+* Debezium has some recommendations regarding MySQL setup:
+
+  https://debezium.io/documentation/reference/stable/connectors/mysql.html#setting-up-mysql
+
+* As well as provides an example configuration on how to configure:
+
+  https://github.com/debezium/container-images/blob/main/examples/mysql/2.3/mysql.cnf
+
+
+### AWS Aurora
+
+Aurora is a MySQL database offered by AWS. By default, binlog is not enabled.
+It is possible to enable the binlog via the AWS Console.
+AWS provides instructions how to in the link below.
+Note that for Debezium, the binlog format must be set to ROW.
+
+https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_LogAccess.MySQL.BinaryFormat.html
+
+
+## Step 2: Debezium user
+
+Within your database, you must create a user which has the privileges
+to access the binlog. Note that the binlog is global, and as such contains
+**all row data** across **all tables** across **all databases**.
+
+1. Below is the SQL to create the dedicated Debezium user with the privileges.
+   **Be sure to fill in your own secure and unique username and password.**
+
+   ```sql
+   CREATE USER 'chosen_debezium_username' IDENTIFIED BY 'chosen_debezium_password';
+   GRANT SELECT, RELOAD, SHOW DATABASES, REPLICATION SLAVE, REPLICATION CLIENT, LOCK TABLES ON *.* TO 'debezium';
+   ```
+
+2. We will refer to the chosen username/password as the Debezium username/password.
+
+
+## Step 3: Create Kafka Connect input connector
+
+Within your Feldera Cloud, you must have already set up a Kafka and
+Kafka Connect (with the Debezium plug-in) instance.
+We will now connect the MySQL database with Kafka Connect.
+
+1. Collect the following information:
+
+    * `[KAFKA CONNECT HOSTNAME:PORT]` : The Kafka Connect hostname and port separated by a colon.
+    * `[KAFKA HOSTNAME:PORT]` : The Kafka hostname and port separated by a colon.
+    * `[MYSQL HOST NAME]` : The hostname of your MySQL instance.
+    * `[MYSQL PORT]` : The port of your MySQL instance (generally 3306)
+    * `[DEBEZIUM USERNAME]` : The created user's username from Step 2 (above)
+    * `[DEBEZIUM PASSWORD]` : The created user's password from Step 2 (above)
+    * `[UNIQUE DATABASE SERVER ID]` : Unique identifier of your database server. See also: https://dev.mysql.com/doc/refman/8.0/en/replication-options.html
+    * `[UNIQUE DATABASE SERVER NAME]` : Unique name of your database server which will be prefixed to all Kafka topics.
+    * `[DATABASES TO CONNECT]` : List of databases (within your MySQL database) to connect
+
+2. Creation of the Debezium-MySQL connector is done via an HTTP POST request to the
```

Comment:
Please use active voice whenever possible: "Create a .... using the following `curl` command:..."